### ISSUE TITLE
Custom device id webcam source for hand tracking

### ISSUE THREAD

**Issue by hartonoda** [MIDIBlocks/handsfree#139]
Hi at the moment the the library automatically select the webcam with facingMode 'user' and there is no way to define a custom device with device id, is it possible to update the library so it accept config with custom device Id ?

thanks

**Comment by XAKEPEHOK** [comment]
+1

**Comment by hartonoda** [comment]
@XAKEPEHOK yes I managed to do this in hackish way by switching the video source on update event, but it would be better if its possible since the beginning. I checked the code it should be easy to do it by accepting the dynamic props
